CHANGELOG.md

```diff
@@ -5,13 +5,21 @@
   * A user-friendly description of a new feature. {issue-number}
 
 ## Fixes
- * The field `TypeEnum.Totp` now points to the correct value.  {#40}
```

Comment:
What happened here?

Reply:
That is an unreleased change that I removed from the changelog. Oops.